From 63e851a98745ba108761ad7ff8d8d61a93a327ad Mon Sep 17 00:00:00 2001
From: guoyi <gy@rock-chips.com>
Date: Fri, 27 Mar 2015 15:33:38 +0800
Subject: [PATCH] rk3368: efuse: fix warning and 	renamed
 rockchip_efuse_get_temp to rockchip_efuse_get_temp_adjust;

Signed-off-by: guoyi <gy@rock-chips.com>
---
 arch/arm/mach-rockchip/efuse.c     | 4 ++--
 arch/arm/mach-rockchip/efuse.h     | 2 +-
 drivers/thermal/rockchip_thermal.c | 2 +-
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/mach-rockchip/efuse.c b/arch/arm/mach-rockchip/efuse.c
index 35fb4dec0cae..b979e411dcd4 100644
--- a/arch/arm/mach-rockchip/efuse.c
+++ b/arch/arm/mach-rockchip/efuse.c
@@ -144,7 +144,7 @@ int secure_regs_wr_32(u64 addr_phy, u32 val)
 	return invoke_regs_wr_fn(PSCI_OS_ACCESS_REG, val_64, addr_phy, SEC_REG_WR_32);
 }
 
-static int __init rk3368_efuse_readregs(u32 addr, u32 length, u8 *buf)
+static int rk3368_efuse_readregs(u32 addr, u32 length, u8 *buf)
 {
 	int ret = length;
 		if (!length)
@@ -314,7 +314,7 @@ int rockchip_get_leakage(int ch)
 	}
 	return 0;
 }
-int rockchip_efuse_get_temp(int ch)
+int rockchip_efuse_get_temp_adjust(int ch)
 {
 	int ret = 0;
 	int temp = 0;
diff --git a/arch/arm/mach-rockchip/efuse.h b/arch/arm/mach-rockchip/efuse.h
index 01c894c6a9b0..96193114aaa9 100644
--- a/arch/arm/mach-rockchip/efuse.h
+++ b/arch/arm/mach-rockchip/efuse.h
@@ -28,5 +28,5 @@
 int rockchip_efuse_version(void);
 int rockchip_process_version(void);
 int rockchip_get_leakage(int ch);
-int rockchip_efuse_get_temp(int ch);
+int rockchip_efuse_get_temp_adjust(int ch);
 #endif
diff --git a/drivers/thermal/rockchip_thermal.c b/drivers/thermal/rockchip_thermal.c
index 7ee48169cafa..1532783c2f4b 100755
--- a/drivers/thermal/rockchip_thermal.c
+++ b/drivers/thermal/rockchip_thermal.c
@@ -1041,7 +1041,7 @@ static int rockchip_thermal_probe(struct platform_device *pdev)
 			error);
 		goto err_disable_pclk;
 	}
-	thermal->cpu_temp_adjust = rockchip_efuse_get_temp(0);
+	thermal->cpu_temp_adjust = rockchip_efuse_get_temp_adjust(0);
 	if (thermal->chip->mode == TSADC_AUTO_MODE)
 	{
 		thermal->chip->initialize(thermal->regs, thermal->tshut_polarity);
-- 
2.35.3

